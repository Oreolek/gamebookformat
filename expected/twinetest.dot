digraph gamebook {
Twine 21->3
1->2
3->2
}
