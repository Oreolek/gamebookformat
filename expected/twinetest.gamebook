title = Gamebookformat Twine 2 Export Example
author = Pelle Nilsson

* 1 start
This is just an experiment exporting to the markup used in [b]Twine 2[/b] (see http://twinery.org).
A known problem is that new-lines are significant in Twine, while they are happily ignored by
gamebookformat. Have to add some hack to strip unintended whitespace from files when exporting
to Twine 2. Also the book has to start at section 1 because currently the startnode given in
the generated file is hardcoded to 1.
Many things like item management and counters are not yet implemented, but from a quick look
int the Twine 2 documentation it should be possible to add all that later.
Creating a Twine 2 document is done using html output format with the -o twine2 option.
Use the "Import From File" link in the Twine 2 web GUI to import the generated HTML file.
From here you can go to [[other]] or [[ending][the end]].

* other
This is another section, just to try references. You can go on to the end at [[ending]].

* ending
It ends here.