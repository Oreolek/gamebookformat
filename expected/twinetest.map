{"start": 1, "other": 3, "ending": 2}