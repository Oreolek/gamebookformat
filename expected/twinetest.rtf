{\rtf1\ansi\ansicpg1252\cocoartf1038\cocoasubrtf360
{\fonttbl\f0\fswiss\fcharset0 Helvetica;}
{\colortbl;\red255\green255\blue255;}
\paperw11900\paperh16840\margl1440\margr1440\vieww14140\viewh14860\viewkind0
\pard\tx566\tx1133\tx1700\tx2267\tx2834\tx3401\tx3968\tx4535\tx5102\tx5669\tx6236\tx6803\ql\qnatural\pardirnatural

\f0\b\fs24 \cf0

\b \qc Gamebookformat Twine 2 Export Example
\b0\
\b Turn to 1 to begin.
\b0\
\
\b \qc {\*\bkmkstart s1}1{\*\bkmkend s1}
\b0\
\ql  This is just an experiment exporting to the markup used in Twine 2 (see http://twinery.org). A known problem is that new-lines are significant in Twine, while they are happily ignored by gamebookformat. Have to add some hack to strip unintended whitespace from files when exporting to Twine 2. Also the book has to start at section 1 because currently the startnode given in the generated file is hardcoded to 1. Many things like item management and counters are not yet implemented, but from a quick look int the Twine 2 documentation it should be possible to add all that later. Creating a Twine 2 document is done using html output format with the -o twine2 option. Use the "Import From File" link in the Twine 2 web GUI to import the generated HTML file. From here you can go to \b {\field{\*\fldinst  REF s3 \\h }{\fldrslt 3}}
\b0
 or \b {\field{\*\fldinst  REF s2 \\h }{\fldrslt the end}}
\b0

.  \
\

\b \qc {\*\bkmkstart s2}2{\*\bkmkend s2}
\b0\
\ql  It ends here. \
\

\b \qc {\*\bkmkstart s3}3{\*\bkmkend s3}
\b0\
\ql  This is another section, just to try references. You can go on to the end at \b {\field{\*\fldinst  REF s2 \\h }{\fldrslt 2}}
\b0
.  \
\

}
