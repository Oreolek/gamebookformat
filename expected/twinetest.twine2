<tw-storydata name="Gamebookformat Twine 2 Export Example" startnode="1" creator="Twine" creator-version="2.0" format="Harlowe" options="">
<style role="stylesheet" id="twine-user-stylesheet" type="text/twine-css"></style><script role="script" id="twine-user-script" type="text/twine-javascript"></script>
<tw-passagedata pid="1" name="1" tags="" position="10,10"> This is just an experiment exporting to the markup used in <b>Twine 2</b> (see http://twinery.org). A known problem is that new-lines are significant in Twine, while they are happily ignored by gamebookformat. Have to add some hack to strip unintended whitespace from files when exporting to Twine 2. Also the book has to start at section 1 because currently the startnode given in the generated file is hardcoded to 1. Many things like item management and counters are not yet implemented, but from a quick look int the Twine 2 documentation it should be possible to add all that later. Creating a Twine 2 document is done using twine2 output format. Use the "Import From File" link in the Twine 2 web GUI to import the generated file. From here you can go to [[3]]
 or [[the end->2]]
. </tw-passagedata>
<tw-passagedata pid="2" name="2" tags="" position="20,20"> It ends here.</tw-passagedata>
<tw-passagedata pid="3" name="3" tags="" position="30,30"> This is another section, just to try references. You can go on to the end at [[2]]
. </tw-passagedata>
</tw-storydata>
