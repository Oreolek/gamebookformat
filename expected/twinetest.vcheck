examples/twinetest.json: ending (death?) section found: ending
