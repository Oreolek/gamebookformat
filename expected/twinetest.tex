\documentclass[a5paper,onecolumn]{book}

\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage[hidelinks]{hyperref}
\usepackage{graphicx}

\usepackage[top=3.3cm, bottom=3.3cm, left=2cm, right=2cm]{geometry}
\newif\ifpdf
\ifx\pdfoutput\undefined
  \pdffalse
\else
  \ifnum\pdfoutput=1
    \pdftrue
  \else
    \pdffalse
  \fi
\fi

\title{Gamebookformat Twine 2 Export Example}
\author{Pelle Nilsson}
\date{}

\newcounter{sectionnr}

\begin{document}

\maketitle

\clearpage

\thispagestyle{empty}

\pagestyle{empty}

Turn to 1 to begin.
\phantomsection
\refstepcounter{sectionnr}
\label{section1}
\subsection*{\begin{center} \textbf{1} \end{center}}

 \noindent
 This is just an experiment exporting to the markup used in Twine 2 (see http://twinery.org). A known problem is that new-lines are significant in Twine, while they are happily ignored by gamebookformat. Have to add some hack to strip unintended whitespace from files when exporting to Twine 2. Also the book has to start at section 1 because currently the startnode given in the generated file is hardcoded to 1. Many things like item management and counters are not yet implemented, but from a quick look int the Twine 2 documentation it should be possible to add all that later. Creating a Twine 2 document is done using html output format with the -o twine2 option. Use the "Import From File" link in the Twine 2 web GUI to import the generated HTML file. From here you can go to \textbf{\autoref{section3}} or the end (\textbf{\autoref{section2}})
. 
\vspace{1em}
\phantomsection
\refstepcounter{sectionnr}
\label{section2}
\subsection*{\begin{center} \textbf{2} \end{center}}

 \noindent
 It ends here.
\vspace{1em}
\phantomsection
\refstepcounter{sectionnr}
\label{section3}
\subsection*{\begin{center} \textbf{3} \end{center}}

 \noindent
 This is another section, just to try references. You can go on to the end at \textbf{\autoref{section2}}. 
\vspace{1em}
\end{document}
